test/show_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+require './lib/show'
+
+class ShowTest < Minitest::Test
+
+  def setup
+    @kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+    @michael_knight = Character.new({name: "Michael Knight", actor: "David Hasselhoff", salary: 1_600_000})
+    @knight_rider = Show.new("Knight Rider", "Glen Larson", [@michael_knight, @kitt])
+  end
+
+  def test_attributes
+    p @knight_rider.name
+    p @knight_rider.creator
+    p @knight_rider.characters
+    p @knight_rider.total_salary
+  end
```

Comment:
again, you're not asserting anything in this test, and you don't have a test for `assert_instance_of`, nor a test for the `total_salary` method.